Add lemmas about condFDiv and update blueprint (#12)
RemyDegenne/testing-lower-bounds: blueprint/src/sections/f_divergence.tex

Apply Lemma~\ref{lem:fDiv_nonneg}.
\end{proof}

\begin{lemma}
  \label{lem:condFDiv_const}
  \lean{ProbabilityTheory.condFDiv_const}
  \leanok
  \uses{def:condFDiv, def:fDiv}

>   Let $\mu, \nu$ be measures on $\mathcal X$ such that $D_f(\mu, \nu) \neq -\infty$, let $\xi$ be a finite measure on $\mathcal Y$.

Reviewer: ```suggestion
  Let $\mu, \nu$ be measures on $\mathcal X$, where $\mu$ is finite, and le $\xi$ be a finite measure on $\mathcal Y$.
```
